src/screens/MainScreen/index.js

```diff
@@ -17,11 +17,29 @@ import ActionButton from "../../components/ActionButton";
 import Train from "./components/Train";
 import SizePicker from "./components/SizePicker";
 import AddCartForm from "./components/AddCartForm";
-import { addCart, setupCartUpdateListener } from "../../helpers";
-import { loadAudio, defaultSize } from "../../constants";
+import Timer from "./components/Timer";
+import {
+  addCart,
+  setupCartUpdateListener,
+  getConstants,
+  setupTriggersUpdateListener
+} from "../../helpers";
+import {
+  loadAudio,
+  trainLeavingAudio,
+  defaultLeaveAnimationDuration,
+  defaultTimerDisplayDuration,
+  defaultSize
+} from "../../constants";
 const moment = require("moment");
 
 const audio = new Audio(loadAudio);
+const leaveAudio = new Audio(trainLeavingAudio);
+
+let constants = null;
+getConstants().then(result => {
+  constants = result;
+});
```

Comment:
do you need to specify a `.catch` here?

Reply:
Errors are handled by `getConstants`. I don't think setting `constants = result` alone will ever error lol